.github/workflows/automation-ondemand-tests.yml

```diff
@@ -134,7 +138,7 @@ jobs:
     env:
       CHAINLINK_COMMIT_SHA: ${{ github.sha }}
       CHAINLINK_ENV_USER: ${{ github.actor }}
-      TEST_LOG_LEVEL: info
+      TEST_LOG_LEVEL: debug
```

Comment:
More logs in e2e tests are almost always better if QAs are fine with that

Reply:
@skudasov I agree. But here I think I will go to `info`. I don't want to change Anirudh's log level